src/services/blog/utils.js

```diff
@@ -0,0 +1,31 @@
+import { compile } from 'tempura';
+import { readFileSync } from 'fs';
+import { join } from 'desm';
+import { randomNumber, shuffle } from 'carret';
+
+/**
+ * Render tempura template into HTML string
+ * @returns {any}
+ */
+function renderTemplate() {
+  const file = readFileSync(join(import.meta.url, 'template.hbs'), { encoding: 'utf8' });
+  return compile(file);
+}
```

Comment:
ngga dikasih parameter aja?

```suggestion
function renderTemplate(data) {
  const file = readFileSync(join(import.meta.url, 'template.hbs'), { encoding: 'utf8' });
  return compile(file)(data);
}
```

biar bisa gini
```js
// before
renderTemplate()({ title, body, url }));

// after
renderTemplate({title, body, url});
```